packages/vuetify/src/components/VTabs/VTabs.ts

```diff
@@ -187,7 +191,10 @@ export default baseMixins.extend<options>().extend({
           height: convertToUnit(this.height),
         },
         props: {
+          activationMode: this.activationMode,
           activeClass: this.activeClass,
+          ariaLabel: this.$attrs['aria-label'],
```

Comment:
The idea is to pass along the `aria-label` and `aria-labelledby` from `v-tabs` to `v-tabs-bar` since the user does not have direct access to `v-tabs-bar`. I can make them attributes in `v-tabs-bar` too if that's what you're thinking.